tests/kola/nondestructive/misc.sh

```diff
@@ -34,6 +56,20 @@ fi
 assert_file_has_content err.txt 'Unknown.*command'
 echo "ok error on unknown command"
 
+stateroot=$(dirname $(ls /ostree/deploy/*/var))
+ospath=/org/projectatomic/rpmostree1/${stateroot//-/_}
+# related: https://github.com/coreos/fedora-coreos-config/issues/194
+(export LANG=C.utf8
+ # And for some reason this one is set in kola runs but not interactive shells
+ unset LC_ALL
+ gdbus call \
+  --system --dest org.projectatomic.rpmostree1 \
+  --object-path /org/projectatomic/rpmostree1/fedora_coreos \
+  --method org.projectatomic.rpmostree1.OSExperimental.Moo true > moo.txt
+ od -x moo.txt
```

Comment:
Is anything being done with this or is it just displaying for debugging reasons?  I was just wondering since it wasn't in the original test.

Reply:
Heh so...you're seeing the remnant of a sad/frustrating 20 minute debugging session trying to figure out why this was failing when moved to kola, turns out something in the stack is setting `LC_ALL=C` which was overriding the `LANG=C.UTF-8`.

I still need to chase that down.

I guess for now...let's keep it since it's harmless?